### PR TITLE
Pza 33 login register page refinement

### PR DESCRIPTION
**Pull request by Prashishgt**
This pull request is for Login/Register refinement, 

- Login and register design was referenced from https://representclo.com/account/login

- All code related to previous design was deleted.

### CODE REVIEW

File: src/components/Card/ModalProductCard.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { MouseEvent } from 'react';
+import Button from 'components/button/Button';
+import { athleisurefront } from 'assets/Images';
+
+function ModalProductCard() {
```
Comment (dipeshkumarsah98) [inline src/components/Card/ModalProductCard.tsx:5]: where is cleanup function ?

File: src/components/Title/Collection_TItle.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-function CollectionTitle({ season, title_name }: CollectionProps) {
-  return (
-    <div className="collection_title_container">
-      <h4 className="collection_title">{`${season} / ${title_name} `}</h4>
-    </div>
-  );
-}
-
-export default CollectionTitle;
+import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/Title/Collection_TItle.tsx:1]: why react ?

File: src/components/Title/Title.tsx [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-const Title = function Title({ title }: TitleProp) {
-  return (
-    <div className="title__container">
-      <h3>{title}</h3>
-    </div>
-  );
-};
-
-export default Title;
+import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/Title/Title.tsx:1]: why react ?

File: src/components/drawers/Rightdrawers.tsx [hunk 4]
```diff
@@ -1,10 +1,9 @@
 import * as React from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/drawers/Rightdrawers.tsx:1]: why react ?

File: src/components/drawers/Rightdrawers.tsx [hunk 5]
```diff
@@ -1,10 +1,9 @@
...
+import { ChevronRightIcon } from 'assets/icons';
```
Comment (dipeshkumarsah98) [inline src/components/drawers/Rightdrawers.tsx:2]: import structure ?

File: src/components/form/ShippingForm.tsx [hunk 6]
```diff
@@ -1,26 +1,13 @@
 import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/form/ShippingForm.tsx:1]: why react ?

File: src/components/ui/input/FloatingInputLabel.tsx [hunk 7]
```diff
@@ -1,6 +1,5 @@
 /* eslint-disable react/require-default-props */
 import React, { useState, ChangeEvent, InputHTMLAttributes } from 'react';
```
Comment (dipeshkumarsah98) [inline src/components/ui/input/FloatingInputLabel.tsx:2]: why again ?

File: src/provider/Skeleton/ProductCardSkeleton.tsx [hunk 8]
```diff
@@ -1,16 +1,16 @@
...
-      <Skeleton
-        variant="rectangular"
-        width={250}
-        height={320}
-        animation="wave"
-      />
-    </Stack>
-  );
-}
+import * as React from 'react';
```
Comment (dipeshkumarsah98) [inline src/provider/Skeleton/ProductCardSkeleton.tsx:1]: react ?

File: src/provider/Pagination.tsx [hunk 9]
```diff
@@ -1,30 +1,30 @@
...
-    <Stack spacing={2}>
-      <Pagination
-        count={totalPage} // Total number of pages
-        page={currentPage} // Current active page
-        onChange={handlePageChange} // Event handler for page change
-      />
-    </Stack>
-  );
-}
+import React, { useState } from 'react';
```
Comment (dipeshkumarsah98) [inline src/provider/Pagination.tsx:1]: react ?

File: src/routes/HomeRoutes.tsx [hunk 10]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```
Comment (dipeshkumarsah98) [inline src/routes/HomeRoutes.tsx:1]: why ?

**Review by dipeshkumarsah98 — DISMISSED** [review]
Update the changes and ask for re-review :+1: .

File: src/components/Card/ModalProductCard.tsx (hunk 1 above)
Comment (Prashishgt) [reply src/components/Card/ModalProductCard.tsx:5]: @dipeshkumarsah98 this component is not using useEffect hook so why is cleanup function required in this? it is a bit confusing

Comment (dipeshkumarsah98) [reply src/components/Card/ModalProductCard.tsx:5]: My mistake. I read file name as Only Model and assume something else :smile:

Comment (Prashishgt) [reply src/components/Card/ModalProductCard.tsx:5]: Okie

**Pull request closed without merging**